src/main/java/org/junitpioneer/jupiter/StdInSource.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Provide values that the extension will read instead of reading the
+ * standard input ({@code System.out}).
+ * This should be used with {@link org.junitpioneer.jupiter.StdIOExtension.StdIn}.
```

Comment:
```suggestion
 * Provide values that the {@code StdIOExtension} extension will read instead of reading the
 * standard input ({@code System.in}).
 * This should be used with {@link org.junitpioneer.jupiter.StdIOExtension.StdIn}.
```